keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -511,3 +586,54 @@ are properly reloading tokens by:
 - **Will enabling / using this feature result in non-negligible increase of
   resource usage (CPU, RAM, disk, IO, ...) in any components?** it adds a
   token minting operation in the API server every ~48 minutes for every pod.
+
+### Troubleshooting
+
+The Troubleshooting section currently serves the `Playbook` role. We may
+consider splitting it into a dedicated `Playbook` document (potentially with
+some monitoring details). For now, we leave it here.
+
+- **How does this feature react if the API server and/or etcd is
+  unavailable?**
+
+  - TokenRequest API is unavailable
+  - configmap containing API server CA bundle cannot be created or fetched
+
+* **What are other known failure modes?**
+
+  - failure to issue token via token subresource
+
+    - Detection: check `apiserver_request_total` with labels
+      `group="",version="v2",resource="serviceaccounts",subresource="token"`
+    - Mitigations: resort to legacy secret based token.
```

Comment:
does this mean to disable the BoundServiceAccountTokenVolume feature gate in the kube-apiserver and recreate pods?

Reply:
yes and done